### DISCUSSION TITLE
Free trial days

### DISCUSSION THREAD

**Discussion by nonghinh** [Kyon147/laravel-shopify#150] [Q&A]
I have 1 problem about the free trial date as follows:
When 1 of my app store has 7 days trial, after one day they remove the app and reinstall the app, the trial days are left 6 days _(this package is automatically processed)_, but I want to reset back to 7 days free trial for that store. Is there any option for this problem?

**Comment by Kyon147** [comment]
The trial days as you've said is worked out by the first day they accepted the Shopify billing. You'd need to write your own logic to reset the trial days if the customer has a charge that had trial days that were left over and reset them.
